src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/ConverterValueBinder.cs

```diff
@@ -1,60 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-using Microsoft.Azure.WebJobs.Host.Converters;
-using Microsoft.Azure.WebJobs.Host.Storage.Queue;
-
-namespace Microsoft.Azure.WebJobs.Host.Queues.Bindings
-{
-    internal class ConverterValueBinder<TInput> : IOrderedValueBinder
-    {
-        private readonly IStorageQueue _queue;
-        private readonly IConverter<TInput, IStorageQueueMessage> _converter;
-        private readonly IMessageEnqueuedWatcher _messageEnqueuedWatcher;
-
-        public ConverterValueBinder(IStorageQueue queue, IConverter<TInput, IStorageQueueMessage> converter,
-            IMessageEnqueuedWatcher messageEnqueuedWatcher)
-        {
-            _queue = queue;
-            _converter = converter;
-            _messageEnqueuedWatcher = messageEnqueuedWatcher;
-        }
-
-        public BindStepOrder StepOrder
```

Comment:
This class gets replaced with AsyncCollectorValueProvider, which implements StepOrder. 



---
In reply to: [62361334](https://github.com/Azure/azure-webjobs-sdk/pull/711#discussion_r62361334) [](ancestors = 62361334)